data/lib/compat/compat.lua

```diff
@@ -1006,3 +1006,13 @@ end
 function Guild.removeMember(self, player)
 	return player:getGuild() == self and player:setGuild(nil)
 end
+
+function doTileAddItemEx(pos, uid, flags)
+	flags = flags or 0
+	local tile = Tile(pos)
+	local item = Item(uid)
+	if tile and item then
+		tile:addItemEx(item, flags)
+	end
+	return false
```

Comment:
But it will be checked anyway by the target function, it's not like you do it in Lua so it doesn't happen in C++, that's what I mean.